[2023-06-09 13:51:57,662] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: my_dag.dbt-2 manual__2023-06-09T13:51:50.948754+00:00 [queued]>
[2023-06-09 13:51:57,671] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: my_dag.dbt-2 manual__2023-06-09T13:51:50.948754+00:00 [queued]>
[2023-06-09 13:51:57,671] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2023-06-09 13:51:57,671] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2023-06-09 13:51:57,671] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2023-06-09 13:51:57,684] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): dbt-2> on 2023-06-09 13:51:50.948754+00:00
[2023-06-09 13:51:57,688] {standard_task_runner.py:52} INFO - Started process 6125 to run task
[2023-06-09 13:51:57,690] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'dbt-2', 'manual__2023-06-09T13:51:50.948754+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp0lbhfy5o', '--error-file', '/tmp/tmpt_7y3swa']
[2023-06-09 13:51:57,691] {standard_task_runner.py:77} INFO - Job 32: Subtask dbt-2
[2023-06-09 13:51:57,740] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.dbt-2 manual__2023-06-09T13:51:50.948754+00:00 [running]> on host d78e0722255b
[2023-06-09 13:51:57,800] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=dbt-2
AIRFLOW_CTX_EXECUTION_DATE=2023-06-09T13:51:50.948754+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-09T13:51:50.948754+00:00
[2023-06-09 13:51:57,801] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-06-09 13:51:57,802] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'dbt run --models dbt2 --profiles-dir /opt/***/dbt/part3/ --project-dir /opt/***/dbt/part3/']
[2023-06-09 13:51:57,816] {subprocess.py:85} INFO - Output:
[2023-06-09 13:52:00,353] {subprocess.py:89} INFO - [0m13:52:00  Running with dbt=1.5.1
[2023-06-09 13:52:00,467] {subprocess.py:89} INFO - [0m13:52:00  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-09 13:52:00,467] {subprocess.py:89} INFO - There are 1 unused configuration paths:
[2023-06-09 13:52:00,467] {subprocess.py:89} INFO - - models.part3.example
[2023-06-09 13:52:00,479] {subprocess.py:89} INFO - [0m13:52:00  Found 2 models, 0 tests, 0 snapshots, 0 analyses, 307 macros, 0 operations, 0 seed files, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-09 13:52:00,481] {subprocess.py:89} INFO - [0m13:52:00
[2023-06-09 13:52:00,537] {subprocess.py:89} INFO - [0m13:52:00  Concurrency: 1 threads (target='dev')
[2023-06-09 13:52:00,537] {subprocess.py:89} INFO - [0m13:52:00
[2023-06-09 13:52:00,543] {subprocess.py:89} INFO - [0m13:52:00  1 of 1 START sql view model public.dbt2 ........................................ [RUN]
[2023-06-09 13:52:00,646] {subprocess.py:89} INFO - [0m13:52:00  1 of 1 OK created sql view model public.dbt2 ................................... [[32mCREATE VIEW[0m in 0.10s]
[2023-06-09 13:52:00,754] {subprocess.py:89} INFO - [0m13:52:00
[2023-06-09 13:52:00,755] {subprocess.py:89} INFO - [0m13:52:00  Finished running 1 view model in 0 hours 0 minutes and 0.27 seconds (0.27s).
[2023-06-09 13:52:00,763] {subprocess.py:89} INFO - [0m13:52:00
[2023-06-09 13:52:00,764] {subprocess.py:89} INFO - [0m13:52:00  [32mCompleted successfully[0m
[2023-06-09 13:52:00,764] {subprocess.py:89} INFO - [0m13:52:00
[2023-06-09 13:52:00,765] {subprocess.py:89} INFO - [0m13:52:00  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2023-06-09 13:52:01,516] {subprocess.py:93} INFO - Command exited with return code 0
[2023-06-09 13:52:01,547] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=dbt-2, execution_date=20230609T135150, start_date=20230609T135157, end_date=20230609T135201
[2023-06-09 13:52:01,601] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-06-09 13:52:01,661] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
